mica-search/src/main/java/org/obiba/mica/search/queries/rql/RQLQueryBuilder.java

```diff
@@ -90,7 +90,7 @@ public TargetQueryBuilder exists(String field) {
     }
 
     public TargetQueryBuilder match(String query) {
-      target.createChildNode(RQLNode.MATCH.name().toLowerCase(), Arrays.asList(query));
+      target.createChildNode(RQLNode.DOC_MATCH.name().toLowerCase(), Arrays.asList(query));
```

Comment:
Why replacing? RQLNode.MATCH still exists.

Reply:
Or name the method accordingly. This is really confusing having these 2 match functions.